sdk/purview/Azure.Analytics.Purview.DataMap/src/Generated/AtlasBusinessMetadataDef.cs

```diff
@@ -0,0 +1,124 @@
+        public AtlasBusinessMetadataDef()
+        {
+            Options = new ChangeTrackingDictionary<string, string>();
+            AttributeDefs = new ChangeTrackingList<AtlasAttributeDef>();
+        }
+
+        /// <summary> Initializes a new instance of <see cref="AtlasBusinessMetadataDef"/>. </summary>
+        /// <param name="category"> The enum of type category. </param>
+        /// <param name="createTime"> The created time of the record. </param>
+        /// <param name="createdBy"> The user who created the record. </param>
+        /// <param name="dateFormatter"> The date format. </param>
+        /// <param name="description"> The description of the type definition. </param>
+        /// <param name="guid"> The GUID of the type definition. </param>
+        /// <param name="name"> The name of the type definition. </param>
+        /// <param name="options"> The options for the type definition. </param>
+        /// <param name="serviceType"> The service type. </param>
+        /// <param name="typeVersion"> The version of the type. </param>
+        /// <param name="updateTime"> The update time of the record. </param>
+        /// <param name="updatedBy"> The user who updated the record. </param>
+        /// <param name="version"> The version of the record. </param>
+        /// <param name="lastModifiedTS"> ETag for concurrency control. </param>
+        /// <param name="attributeDefs"> An array of attribute definitions. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal AtlasBusinessMetadataDef(TypeCategory? category, long? createTime, string createdBy, AtlasDateFormat dateFormatter, string description, string guid, string name, IDictionary<string, string> options, string serviceType, string typeVersion, long? updateTime, string updatedBy, long? version, string lastModifiedTS, IList<AtlasAttributeDef> attributeDefs, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            Category = category;
+            CreateTime = createTime;
+            CreatedBy = createdBy;
+            DateFormatter = dateFormatter;
+            Description = description;
+            Guid = guid;
+            Name = name;
+            Options = options;
+            ServiceType = serviceType;
+            TypeVersion = typeVersion;
+            UpdateTime = updateTime;
+            UpdatedBy = updatedBy;
+            Version = version;
+            LastModifiedTS = lastModifiedTS;
+            AttributeDefs = attributeDefs;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> The enum of type category. </summary>
+        public TypeCategory? Category { get; set; }
+        /// <summary> The created time of the record. </summary>
+        public long? CreateTime { get; set; }
```

Comment:
```suggestion
        public DateTimeOffset? CreateTime { get; set; }
```

Reply:
![image](https://github.com/Azure/azure-sdk-for-net/assets/78060542/e353d016-3b95-4471-b798-7af414ece64d)
This is the discussion in the typespec PR why we use long instead of DataTimeOffset.
https://github.com/Azure/azure-rest-api-specs/pull/26702#issuecomment-1925645594
https://github.com/Azure/azure-rest-api-specs/pull/26702#issuecomment-1928546157